src/main/java/com/aws/greengrass/clientdevices/auth/iot/Certificate.java

```diff
@@ -36,10 +37,12 @@ public class Certificate implements AttributeProvider {
 
     public enum Status {
         ACTIVE,
-        UNKNOWN
+        UNKNOWN,
+        TRUST_EXPIRED
```

Comment:
We already have `UNKNOWN` - just use that?

Reply:
I have recommended @vaibhavmurkute to do this. @jbutler the reasoning is that UNKNOW does not really represent what state the certificate is in. We actually know what happened which represents a valid state in the domain